cypress/integration/cucumber/step_definitions/refapp-3.x/01-login/patient-search.js

```diff
@@ -0,0 +1,36 @@
+import {Given, After, Before} from 'cypress-cucumber-preprocessor/steps';
+
+let patient_uuid = null;
+
+Before({tags: '@patient-involved'}, () => {
+    cy.createPatient().then((user) => {
+        patient_uuid = user.uuid;
+    });
+});
```

Comment:
Its like patient creation doesn't occur.  could it be related to https://github.com/openmrs/openmrs-test-3refapp/pull/3 

cc @jayasanka-sack @ibacher @pasindur99

Reply:
I don't think this issue is related to the data mutation issue.
In this case the issue is related with the wrong patient name (I guess) - 
https://github.com/openmrs/openmrs-test-3refapp/runs/7275918359?check_suite_focus=true
<img width="876" alt="image" src="https://user-images.githubusercontent.com/77311602/178199002-ba3fb243-34d2-4275-8109-4cca261e521a.png">

Try renaming the patient's name to **John Lee**

Reply:
You can create another folder for the patient-search.js step definition file as we did before in the following image.
<img width="293" alt="image" src="https://user-images.githubusercontent.com/77311602/178199736-b6a3415a-bc20-4782-9f93-94213a2e695d.png">

Reply:
We might need to update the following function,
https://github.com/openmrs/openmrs-test-3refapp/blob/a59f83c331036894ca3de5d1e434db6bf28adcbd/cypress/support/commands.js#L301

Reply:
Ooohhh!
I forgot to say but this worked fine on local